modal_popup.js

```diff
@@ -0,0 +1,195 @@
+// const viewProject = document.querySelectorAll(".btn").length;
```

Comment:
- [ ] **[OPTIONAL]:** Remember to delete all commented code! It makes your codebase look ugly and could potentially affect the readability and maintainability of the code in the future! I would suggest you fix this. 🤓 🛠️